tests/unit-tests/ambiguous/__snapshots__/arrow-parens-avoid.test.ts.snap

```diff
@@ -6,7 +6,7 @@ exports[`ambiguous > config > arrowParens: "avoid" > (oh, no) => {} > with semi,
   module top level component. Explicit default export module top level
   component. Explicit default export module top level component. Explicit
   default export module top level component.
-</template>;
```

Comment:
```js
function template() {}
(oh, no) => {};
```
is without semi
https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AE8AtgA4A2AhvABQCU+wAvgDpTUQAWANPlBPQF4AfA0YBuViC4gIJXNADOyUBQBOqiAHcACmoRKUFMpooBPJdIBGqimADWcGAGUKROABkcUOMlRGFcFY29o5OJLZeAObIMKrogSABRDgxcQlwAB4kcKo4brBGACo5UGo4cAZ+ZAHSClFkcACK6BDwvv4JAFYKGU71TS1tSFU1IACOg3DaGiQGIBQKALTecAAma1IgsRQ4ZFEAwhBERBTI82Rkm3VQkQ0AgjCxOJbo8No5nt7t1QkcMERkADqHBw8AU4TAcCc+lBOAAbqDTGcwAoLCA4fEAJJQdawJxgXJyO44pwwUwNb6jEgaAKAmwkM7Uio5OE+aReAKqGDTCiRE6UhLhVScs5k7IKAk4OSbaleGCAnCrGAcZAADgADNJVHAJjhtTy+adhh1pDAKJYFUqVUgAEzSdABQrmyomkBwIiWNbrVbuCg3dC8uAAMQgqhOjyiZworwgIEYjCAA